x-pack/plugins/cases/server/client/client.ts

```diff
@@ -13,20 +12,32 @@ import { UserActionsSubClient, createUserActionsSubClient } from './user_actions
 import { CasesClientInternal, createCasesClientInternal } from './client_internal';
 import { createSubCasesClient, SubCasesClient } from './sub_cases/client';
 import { ENABLE_CASE_CONNECTOR } from '../../common/constants';
+import { ConfigureSubClient, createConfigurationSubClient } from './configure/client';
+import { createReportersSubClient, ReportersSubClient } from './reporters/client';
+import { createStatusStatsSubClient, StatusStatsSubClient } from './status_stats/client';
+import { createTagsSubClient, TagsSubClient } from './tags/client';
 
 export class CasesClient {
   private readonly _casesClientInternal: CasesClientInternal;
   private readonly _cases: CasesSubClient;
   private readonly _attachments: AttachmentsSubClient;
   private readonly _userActions: UserActionsSubClient;
   private readonly _subCases: SubCasesClient;
+  private readonly _configure: ConfigureSubClient;
+  private readonly _reporters: ReportersSubClient;
+  private readonly _statusStats: StatusStatsSubClient;
+  private readonly _tags: TagsSubClient;
 
   constructor(args: CasesClientArgs) {
     this._casesClientInternal = createCasesClientInternal(args);
     this._cases = createCasesSubClient(args, this, this._casesClientInternal);
     this._attachments = createAttachmentsSubClient(args, this._casesClientInternal);
     this._userActions = createUserActionsSubClient(args);
     this._subCases = createSubCasesClient(args, this);
+    this._configure = createConfigurationSubClient(args, this._casesClientInternal);
+    this._reporters = createReportersSubClient(args);
+    this._statusStats = createStatusStatsSubClient(args);
```

Comment:
I like the idea of having a dedicated KPI sub client. I think it will be much readable if the user can use the API as `casesClient.stats.getAll` instead of `statusStats`. In the feature, we may want to add more stats. For example if we add telemetry.